packages/api/src/routes/medical/document.ts

```diff
@@ -145,21 +142,20 @@ async function getDownloadUrl(req: Request): Promise<string> {
 
   const fileName = getFromQueryOrFail("fileName", req);
   const fileNameString = docFileNameSchema.parse(fileName);
-  const fileHasCxId = fileNameString.includes(cxId);
+  const fileHasCxId = fileNameString.includes(cxId); // Should probably check for startsWith
```

Comment:
This makes the logic kinda brittle though, since we could  change the way we name the files and we would not realize we had broken this right?

Reply:
maybe we could put this logic in a small helper function in the same file as 
`createDocumentFilePath` which I think is the function that creates document paths?

Reply:
> This makes the logic kinda brittle though, since we could change the way we name the files and we would not realize we had broken this right?

Yeah, that's why I suggested on https://github.com/metriport/metriport-internal/issues/2422 that we move the param to indicate the file to be the doc ref ID instead of the file name.

I think we leave it as is now (I can remove the comment) and we try to prioritize changing the endpoint?

Reply:
removed